be/src/util/threadpool.cpp

```diff
@@ -610,6 +607,47 @@ void ThreadPool::check_not_pool_thread_unlocked() {
     }
 }
 
+Status ThreadPool::set_min_threads(int min_threads) {
+    if (min_threads <= _max_threads) {
+        _min_threads = min_threads;
```

Comment:
Even though `_min_threads` is type of `AtomicInt32`, this function is not thread safe.

Reply:
> Even though `_min_threads` is type of `AtomicInt32`, this function is not thread safe.

You're right, thank you.